omero/sysadmins/unix/install-web.txt

```diff
@@ -29,9 +29,9 @@ There are a number of good ways to easily deploy OMERO.web.
 
 Deploying OMERO.web separately from OMERO.server is recommended as they
 perform best under different circumstances and require a different set of
-dependencies. Please consider using :doc:`OMERO.web trial deployment <install-web/install-web-trial>`.
+dependencies. Please consider using :doc:`OMERO.web deployment <install-web/web-deployment>`.
```

Comment:
This doc link needs updating to 'OMERO.web WSGI deployment' (or just let it automatically use the page title for the link) because otherwise we have a lot of different 'deployment' sections/titles with no clear differentiation.

In fact, it might also be good to retitle this page as 'Installing OMERO.web' or 'Setting up OMERO.web'